src/CNGUTS.CPP

```diff
@@ -30,7 +30,6 @@
 #include "cse.h"
 #include <impf.h>	// impfStart impfIvl 2-94
 #ifdef WINorDLL
-#include <windows.h>		// probably needed before brfw.h
```

Comment:
Possibly leave this one also.  Again, it is not being compiled until WINorDLL is revived.

Reply:
Done. I don't understand why is GitHub not updating it.